### PR TITLE
OCM-8418 | test: update readme for automation labels usage

### PR DESCRIPTION
**Pull request by yasun1**
Automation labels Docs: https://docs.google.com/document/d/1R7ZC9vJ6FsXOBA3zlQVA5jrF9hlBuwWViZMo5AOxPWk/edit

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -69,12 +69,14 @@ var _ = Describe("Create Machine Pool", func() {
     `<card id> | test: automated cases id:123456,123457`
 
 ### Labels
+Label Design Doc: [ROSA CLI Automation labels](https://docs.google.com/document/d/1R7ZC9vJ6FsXOBA3zlQVA5jrF9hlBuwWViZMo5AOxPWk/edit)
```
Comment (radtriste) [inline tests/README.md:72]: I cannot access that file if I am not Red Hat. Either make it available to everybody or remove the link ;)

**Review by radtriste — APPROVED** [review]

File: tests/README.md (hunk 1 above)
Comment (yasun1) [reply tests/README.md:72]: Good catch. Let us remove the link for the doc is internal. We need an individual doc for the label introduction.

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**